include/net/lwm2m_client_utils_location.h

```diff
@@ -105,6 +108,14 @@ int location_assistance_pgps_request_send(struct lwm2m_ctx *ctx);
  */
 int location_assistance_init_resend_handler(void);
 
+/**
+ * @brief Initialize the location assistance library and disable resend handler.
+ *
+ * @return Returns a negative error code (errno.h) indicating
+ *         reason of failure or 0 for success.
+ */
+int location_assistance_disable_resend(void);
```

Comment:
I don't understand..
Why is this a public API?

When do I have to call this?
If there is fixed amount of retries, why doesn't the library automatically stop?

Reply:
This just disable Re send feature and also temporary error re-send. I will make a internal API. This is just test more about test purpose. This disable activated re-send functionality